files/galaxy/tpv/tools.yml

```diff
@@ -1006,6 +1006,15 @@ tools:
       require:
         - singularity
 
+  toolshed.g2.bx.psu.edu/repos/galaxy-tools/braker3/braker3/.*:
+    mem: 4
+    cores: 6
+    params:
+      singularity_container_id_override: docker://teambraker/braker3:v.1.0.4
```

Comment:
@rlibouba I don't know if it works with the `docker:` prefix. Is that what is used in TPV usegalaxy.fr as well?

We can merge and test I think.

Reply:
we use the docker:// with the old sorting hat, it works fine

Reply:
I tested BRAKER3 on UseGalaxy.eu, and the tests failed. The error reads "/gmes/gmes_petap.pl: No such file or directory".

Reply:
oh maybe it's not using the singularity image?

Reply:
I think its the wrong tool ID